internal/services/media/media_streaming_endpoint_resource.go

```diff
@@ -137,6 +137,23 @@ func resourceMediaStreamingEndpoint() *pluginsdk.Resource {
 				},
 			},
 
+			"current_sku": {
```

Comment:
Can the sku not be set?

Reply:
<img width="777" alt="image" src="https://user-images.githubusercontent.com/104055472/220622450-07110158-f9bb-4408-b630-6de85f797e73.png">

Reply:
changed to `sku`